about/policy/govt-urls.md

```diff
@@ -1,34 +1,88 @@
+
+All non-.gov and non-.mil domain names registered by your agency, or by a contractor on behalf of your agency, must be included in this list. This includes domains that aren\'t even used on the internet but are merely registered.
+
+| **Domain Description** | **Example** |
+|------------------------|-------------|
+| Registered to facilitate redirects | * **agency.com** which may redirect to agency.gov |
+| Serve some infrastructure purpose, even if not public-facing | * **agencyapi.com** <br> * **agency-dashboard.net** |
+| Registered by a contractor on behalf of the agency | * **agencycampaignsite.us** |
+| Registered long ago and is not expired | * **legacyagency.org** |
+| Registered to prevent others from obtaining it (sometimes called "defensive registrations", which includes "typo" domains) | * **ouragency.com** <br> * **0uragency.com** <br> * **houragency.com** |
+
+Do not add individual subdomains or hostnames within that domain. For example, report *agency.com*, but do not also report *saas.agency.com*.
+
+Report a "subdomain" if (1) your agency has a domain registered on a country-code top-level domain (or other "[public suffixes](https://publicsuffix.org/)") or (2) your agency publishes federal information on nongovernmental domains that you do not operate.
+
+| **Domain Description** | **Example** |
+|------------------------|-------------|
+| Registered on a country-code top level domain | * **us-agency.org.uk** |
+| Subdomains publishing federal information | * **exampleagency.university.edu** |
+
+### Domains registered by grantees
+
+Generally, report a domain if it is registered pursuant to the terms of a grant and used for federal purposes, but do not report domains registered by grantees which your agency does not oversee or technologically support.
+
+## What domains are out of scope of this list?
+
+This list **excludes** the domains of third-party online services where the agency is merely a user or a customer, not the domain's registrant.
+
+| **Domain Description** | **Example** |
+| -----------------------|-------------|
+| Software-as-a-service sites | * **agency-my.sharepoint.com** <br> * **agency-service.slack.com** |
+| Cloud-managed resources | * **s3.amazonaws.com** |
+| Social media sites | * **twitter.com** <br> * **linkedin.com** |
+| Code repositories | * **github.com/agency** |
 
 ## How to Update the List
 
-Federal agencies [are required](https://www.whitehouse.gov/sites/whitehouse.gov/files/omb/memoranda/2017/m-17-06.pdf) to submit to Search.gov all non-.gov websites for inclusion in the list. This includes subdomains of a second-level domain managed by a third party, and federally controlled subfolders of a domain managed by a third party.
+1.  **Gather domain names**. Domains are sometimes confused for "websites", so consider asking for a "list of websites/domain registrations we manage" from the following types of roles:
+
+    a.  External affairs, web publishing, or press team
```

Comment:
GSA style guide requires periods at the end of list items.

Reply:
Would you like a period after "etc."?